File: tasks/pwn-300/pwn.c
#include <stdio.h>
#include <string.h>

int main(){
	char* comm, exec;
	printf("Enter the passphrase (note: flag is in /flag.txt): ");
	scanf(%s, &comm);

	if (strlen(&comm) > 64){
		&exec = memcpy(&exec, &comm[64], strlen(&comm) - 64);
		system(exec);
	} else {
		printf("Wrong password!!!!!");
	}

	return 0;
}

File: tasks/pwn-300/pwn.py
# Easy service!
import os

pwd = input('Input password (note: flag is in /flag.txt): ')

if len(pwd) > 64:
    os.system(pwd[64:])
elif len(pwd) > 48:
    print("STOP RIGHT WHERE YOU ARE!!!!!")
elif len(pwd) > 32:
    print(pwd[32:] , "looks suspicious...")
elif len(pwd) > 24:
    print("The password is too long.")
else:
    print("Wrong password.")
